EduVPN/ViewModels/SearchViewModel.swift

```diff
@@ -184,7 +184,17 @@ private extension SearchViewModel {
         let hasTwoOrMoreDots = searchQuery.filter { $0 == "." }.count >= 2
         if hasTwoOrMoreDots {
             var url = searchQuery.lowercased().trimmingCharacters(in: .whitespacesAndNewlines)
-            if !url.hasPrefix("https://") {
+            if url.hasPrefix("https://") {
```

Comment:
Not blocking, but why not just look at what `url.scheme` returns instead of custom parsing?

Reply:
In the issue (#407), @jeroenleenarts had the same question. When we give a scheme-less string (like "vpn.tuxed.net") to URLComponents or URL, it considers that whole string to be a path (we want it to be seen as a host), so we have it as a string to keep handling of all scenarios simple.